elastio-aws-org-elastio-cfn-deployment/README.md

```diff
@@ -0,0 +1,94 @@
+# elastio-aws-org-elastio-cfn-deployment
+
+With this solution you can in a simple way redistribute elastio cfn deployment
+across your org.
+
+Terraform code what implements it deploys a service catalog product, which is
+shared across the org and could be used to deploy elastio cfn stack in exiting
+accounts, also part of the code is a lambda that listens for new account
+creation events and triggers a product provisioning in a new account.
+
+## Requirements
+
+- AWS
+  - Control Tower should be configured along with Landing Zone
+  - Make sure to run this code under credentials of the root aws account
+- Tools
+  - Python 3.6 or newer
+  - zip
+  - terraform 0.13 or newer
+  - aws cli
+
+## Usage
+
+To deploy the solution run the following commands:
+
+```console
+terraform init
+terraform plan
+terraform apply -auto-approve
+```
+
+## Launching the product
+
+_(Both cli commands are supposed to be run under target account where the
```

Comment:
Everywhere you have written `cli` replace with `CLI`.  Abbreviations in English are always ALL CAPS.

Reply:
Replaced